intelmq/bots/BOTS

```diff
@@ -1,5 +1,15 @@
 {
     "Collector": {
+
+        "bitsight":{
+        "module.intelmq.bots.collectors.bitsight.collector",
+        "parameters":{
+             "feed":"Bitsight",
+             "http_url": "http://alerts.bitsighttech.com:8080/stream?key=",
+             "api_key": "<insert your api key>",
```

Comment:
To be consistent with other bots, remove this option and add the `<api>` string to the URL